Components/Schedule/meya.schedule.yaml.meta.yaml

```diff
@@ -0,0 +1,45 @@
+---
+docstring: |
+    ## meya.schedule
+    Schedule a flow to run either at set periodic intervals, or at specific times using cron.
+    ### Properties
+    These properties are common to both periodic and cron versions of `meya.schedule`.
+    | Property | Description | Required |
```

Comment:
This syntax isn't rendering as a table.
<img width="645" alt="screen shot 2018-05-08 at 2 45 18 pm" src="https://user-images.githubusercontent.com/14201959/39776309-73cfaa20-52ce-11e8-8154-d551c97b55ed.png">